documentation/Installing-Clear-Containers-on-Ubuntu.md

```diff
@@ -11,14 +11,19 @@ used but some issues may exist.
 This step is optional and required only in the case where docker is not installed
 on the system, or the explicitly supported version of docker is required.
 
+Note that `xenial` (Ubuntu 16.04) is specified below. This is required
+for both Ubuntu 16.04 *and* Ubuntu 16.10 (`yakkety`) since the Clear
+Containers supported version of Docker is not available as an Ubuntu
+16.10 package.
+
```

Comment:
Please rephrase paragraph removing ambiguous references. 
My suggestion is: 

> The  `add-apt-repository command` references `ubuntu-xenial`. Both Ubuntu 16.04 and 16.10 require this reference. since the version of Docker supported by Clear Containers is not available as a Ubuntu 16.10 package.